docs/tutorial/grid_comps/code_generator/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+esma_set_this (OVERRIDE MAPL.mycomponent)
+
+set (srcs
+    MyComponent_GridComp.F90
+    )
+
```

Comment:
The directory is not the `code_generator` so maybe come up with a good directory name and make the component agree with that?

"MyComponent" is minimally acceptable but maybe instead ACG_GridComp or some such?

Reply:
Because it is part of the tutorial sample codes, I propose as directory `sample_code_generation`. The name of the grid component will then be `SCG_GridComp`. I want to avoid `ACG` that is already the acronym of the tool. Any thought?

Reply:
Given that `ACG` is a common abbreviation for the automatic code generator, I vote for `automatic_code_generator` for the directory, `ACG_GridComp.F90` for the GC, and `ACG_StateSpecs.rc` for the specs file.